install/kibana-install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: T.H. (ELKozel)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
```

Comment:
missing "core" dependencies.
not adhering to the established patterns and conventions

Reply:
By "core" dependencies, could you elaborate a bit more, if I'm missing any. I see a lot of scripts having:
- curl
- sudo
- mc